app/gui/view/src/project_list.rs

```diff
@@ -196,6 +197,23 @@ mod background {
 
 
 
+// ===========
+// === FRP ===
+// ===========
+
+ensogl::define_endpoints! {
+    Input {
+        /// A list of projects (name, ID) to choose from
```

Comment:
1. Documentation should be real English sentences. Please fix it everywhere in this PR.
2. If this is a "list of projects", why this FRP endpoint is not named "list_projects"? Either the name or documentation is wrong.

Reply:
1. This isn't a sentence, is it?
2. I was mimicking the `set_` style I've seen for other FRP inputs. ([Example](https://github.com/enso-org/enso/blob/develop/lib/rust/ensogl/component/label/src/lib.rs#L69-L70))

Reply:
1. Exactly, it is not, but it should be a sentence.
2. The documentation tells that this FRP stream is "A list of projects". If it is "a list of projects", it should be named "project_list" or something like that. If this FRP stream is not designed to list projects, the the documentation is invalid, as the docs does not tell that this stream is used to set the list of the projects.